source/opcode.cpp

```diff
@@ -612,6 +612,39 @@ bool spvOpcodeIsDebug(SpvOp opcode) {
   }
 }
 
+bool spvOpcodeIsCommutative(SpvOp opcode) {
```

Comment:
How about `spvOpcodeIsCommutativeBinaryOperator` as a name?

Reply:
This name `spvOpcodeIsCommutativeBinaryOperator` is better, since the operator is binary.